### PR TITLE
refactor: cobra.flux_analysis.loopless

### PR DESCRIPTION
**Pull request by synchon**
* [ ] fix #(issue number)
* [x] description of feature/fix
* [x] tests added/passed
* [ ] add an entry to the [next release](../release-notes/next-release.md)

This PR adds typing annotations for `cobra.flux_analysis.loopless` and its unit tests. It also upgrades the code style for them to maintain Python 3.6+ compatibility.

### CODE REVIEW

File: src/cobra/test/test_flux_analysis/test_loopless.py [hunk 1]
```diff
@@ -37,14 +33,14 @@ def construct_ll_test_model():
     scope="function",
     params=[s for s in ["glpk", "cplex", "gurobi"] if s in sutil.solvers],
 )
-def ll_test_model(request):
+def ll_test_model(request: str) -> Model:
```
Comment (Midnighter) [inline src/cobra/test/test_flux_analysis/test_loopless.py]: `request` is not of type string (`request.param` is). I think the type is `pytest.FixtureRequest` but I'm not sure.

File: src/cobra/test/test_flux_analysis/test_loopless.py [hunk 2]
```diff
@@ -37,14 +33,14 @@ def construct_ll_test_model():
...
     """Return test model set with different solvers."""
     test_model = construct_ll_test_model()
     test_model.solver = request.param
     return test_model
 
 
-def test_loopless_benchmark_before(benchmark):
+def test_loopless_benchmark_before(benchmark) -> None:
```
Comment (Midnighter) [inline src/cobra/test/test_flux_analysis/test_loopless.py]: It doesn't improve much but you could annotate `benchmark` as `typing.Callable`.

File: src/cobra/test/test_flux_analysis/test_loopless.py [hunk 3]
```diff
@@ -56,13 +52,13 @@ def _():
     benchmark(_)
 
 
-def test_loopless_benchmark_after(benchmark):
+def test_loopless_benchmark_after(benchmark) -> None:
```
Comment (Midnighter) [inline src/cobra/test/test_flux_analysis/test_loopless.py]: Same as above.

**Review by Midnighter — CHANGES_REQUESTED** [review]
One wrong type annotation and a suggestion for annotating the `benchmark` fixture. Otherwise it looks A-okay.

File: src/cobra/test/test_flux_analysis/test_loopless.py (hunk 1 above)
Comment (synchon) [reply src/cobra/test/test_flux_analysis/test_loopless.py]: Oh yes you are right, I overlooked it.

File: src/cobra/test/test_flux_analysis/test_loopless.py (hunk 2 above)
Comment (synchon) [reply src/cobra/test/test_flux_analysis/test_loopless.py]: I'll add it.

**Review by Midnighter — APPROVED** [review]
👍🏼

**Pull request merged**